pkg/email/plan_discussion_created.go

```diff
@@ -10,7 +12,7 @@ type PlanDiscussionCreatedBodyContent struct {
 	ClientAddress     string
 	DiscussionID      string
 	UserName          string
-	DiscussionContent string
+	DiscussionContent template.HTML
```

Comment:
This allows the template to render the content as html instead of escaped

Reply:
Can you add this as an inline comment on the field?